etc/telegraf.conf

```diff
@@ -973,6 +973,15 @@
 #   #   # Should be set manually to "application/json" for json data_format
 #   #   Content-Type = "text/plain; charset=utf-8"
 #
+#   ## MaxIdleConns controls the maximum number of idle (keep-alive)
+#   ## connections across all hosts. Zero means no limit.
+#   # max_idle_conn = 0
+#
+#   ## MaxIdleConnsPerHost, if non-zero, controls the maximum idle
+#   ## (keep-alive) connections to keep per-host. If zero,
+#   ## DefaultMaxIdleConnsPerHost is used(2).
+#   # max_idle_conn_per_host = 0
+#
```

Comment:
The files in `etc/*` are automatically updated after a PR is merged, os no need to touch these files.